storybook/storybook-react/src/Heading/Heading.docs.mdx

```diff
@@ -28,9 +28,9 @@ import README from "../../../../packages/css/src/components/heading/README.md?ra
 
 <Canvas of={HeadingStories.Heading4} />
 
-### Omgekeerde kleur
+### Inverse colour
```

Comment:
The next syntax uses american english "InverseColor", we should discuss if we're using UK or US english.

Reply:
We use British English for documentation – [policy here](https://amsterdam.nl/stijlweb/taalbeleid/engels-talen/).

We write code in US English to match the web APIs – it doesn’t make much sense to try things like `backgroundColor: backgroundColour`.